src/Annotations/AppKernel.php

```diff
@@ -103,6 +104,7 @@ public function __construct(string $environment = 'dev')
                 'authority' => '',
                 'group' => '',
             ],
+            'html_purifier.cache_dir' => $config['html_purifier']['cache_dir'] ?? __DIR__.'/../../var/html_purifier/cache',
```

Comment:
We need to create the `/srv/annotations/var/html_purifier/cache` in all environments in order for this to pass.